classes/Message.php

```diff
@@ -56,7 +56,7 @@ class MessageCore extends ObjectModel
         'table' => 'message',
         'primary' => 'id_message',
         'fields' => array(
-            'message' =>        array('type' => self::TYPE_STRING, 'validate' => 'isCleanHtml', 'required' => true, 'size' => 1600),
+            'message' =>        array('type' => self::TYPE_STRING, 'validate' => 'isCleanHtml', 'required' => true, 'size' => 6400),
```

Comment:
are you sure the field in database can handle 6400 characters?

Reply:
Oups in my own shop yes 👍 i forgot 1.6.1.18.sql

I create a new one because i do this comit directly on github.com

Reply:
It's a text type

Reply:
It seems MySQL can handle 65536 chars (= 2^16)

https://dev.mysql.com/doc/refman/5.7/en/storage-requirements.html#data-types-storage-reqs-strings